infrastructure/variables.tf

```diff
@@ -58,11 +58,16 @@ variable "jenkins_AAD_objectId" {
   description                 = "(Required) The Azure AD object ID of a user, service principal or security group in the Azure Active Directory tenant for the vault. The object ID must be unique for the list of access policies."
 }
 
-variable "definition_cache_ttl_sec" {
+variable "definition_cache_max_idle_sec" {
   type = "string"
   default = "259200"
 }
 
+variable "definition_latest_version_ttl_sec" {
+  type = "string"
+  default = "0"
```

Comment:
Looking at the documentation it says: 
```Its default value is 0, which means infinite (no expiration and eviction)```
This means by default now we are caching forever, which is not what we want

Reply:
Changed to default 30.